tests/Aspire.Hosting.Elasticsearch.Tests/ElasticsearchPublicApiTests.cs

```diff
@@ -0,0 +1,116 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Xunit;
+
+namespace Aspire.Hosting.Elasticsearch.Tests;
+
+public class ElasticsearchPublicApiTests
+{
+    [Fact]
+    public void AddElasticsearchContainerShouldThrowsWhenBuilderIsNull()
```

Comment:
Sorry my typo. It needs to be fixed
ShouldThrow'S' => ShouldThrow

Same for the code below